index.html

```diff
@@ -8,47 +8,324 @@
       rel="stylesheet"
       href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
     />
+    <title>Desktop version</title>
     <link rel="stylesheet" href="style.css" />
-    <title>Setup and mobile version</title>
   </head>
   <body>
```

Comment:
[OPTIONAL] Please remove space from the right and left of the page and follow the design exactly. Kindly see the below screenshot for reference. 
![space](https://user-images.githubusercontent.com/35504149/130317844-cd14c624-d63b-46b2-8c16-f92a0af72f15.png)